lib/utils/ast.js

```diff
@@ -0,0 +1,62 @@
+function isPageIdentifier({ callee }) {
+  return (
+    callee &&
+    callee.type === 'MemberExpression' &&
+    callee.object.type === 'Identifier' &&
+    callee.object.name === 'page'
+  );
+}
+
+function isEvalIdentifier({ callee }) {
+  return (
+    callee &&
+    callee.property &&
+    callee.property.type === 'Identifier' &&
+    callee.property.name === '$eval'
+  );
+}
```

Comment:
How about something like this?

```suggestion
function isCalleeProperty({ callee }, name) {
  return (
    callee &&
    callee.property &&
    callee.property.type === 'Identifier' &&
    callee.property.name === name
  );
}
```

Then usage would be:

```js
isCalleeProperty(node, '$eval')
```